bench_pytorch/README.md

```diff
@@ -5,29 +5,55 @@
 The implementation of benchmarking [PyTorch](https://github.com/pytorch/pytorch) uses the [Transformers Library by Huggingface](https://github.com/huggingface/transformers) under the hood. The reason being, Transformers provides an easy to use interface for Llama-2-7B model in PyTorch backend.
 
 
-### 🚀 Running the PyTorch Benchmark.
+## 🚀 Running the PyTorch Benchmark.
 
-You can run the PyTorch benchmark using the following command:
+We can run the PyTorch benchmark for two models: [Llama2](https://huggingface.co/meta-llama/Llama-2-7b) and [Mistral-7B v0.1](https://huggingface.co/mistralai/Mistral-7B-v0.1) Here is how we run benchmark for PyTorch.
 
 ```bash
 ./bench_pytorch/bench.sh \
-  --prompt <value> \            # Enter a prompt string
-  --max_tokens <value> \        # Maximum number of tokens to output
-  --repetitions <value> \       # Number of repititions to be made for the prompt.
-  --log_file <file_path> \      # A .log file underwhich we want to write the results.
-  --device <cpu/cuda/metal> \   # The device in which we want to benchmark.
-  --models_dir <path_to_models> # The directory in which model weights are present
+  --prompt <value> \               # Enter a prompt string
+  --max_tokens <value> \           # Maximum number of tokens to output
+  --repetitions <value> \          # Number of repititions to be made for the prompt.
+  --device <cpu/cuda/metal> \      # The device in which we want to benchmark.
+  --model_name <name-of-the-model> # The name of the model. (options: 'llama' for Llama2 and 'mistral' for Mistral-7B-v0.1)
 ```
 
 To get started quickly you can simply run:
 
 ```bash
-./bench_pytorch/bench.sh -d cuda
+./bench_pytorch/bench.sh -d cuda -n llama
 ```
-This will take all the default values (see in the [bench.sh](/bench_pytorch/bench.sh) file) and perform the benchmarks. You can find all the benchmarks results for PyTorch [here](/docs/llama2.md).
 
+This will take all the default values (see in the [bench.sh](/bench_pytorch/bench.sh) file) and do the benchmarks for Llama 2 and Mistral models. You can find all the benchmarks results for Llama 2 7B Chat [here](/docs/llama2.md), and Mistral 7B instruct [here](/docs/mistral.md).
 
-### 👀 Some points to note:
```

Comment:
can you review the notes? I see references to old models (no chat model) for example

Reply:
I updated this section with latest info